_posts/en/newsletters/2020-08-19-newsletter.md

```diff
@@ -0,0 +1,208 @@
+  application for conducting P2P [DLC-based][dlcs] derivatives on Bitcoin regtest
+  and testnet. The application allows the specification of a financial
+  agreement and creation of a corresponding locking transaction of funds between
+  the two parties of that agreement. Upon contract maturity, a price oracle
+  provides a signature for the closing transaction spending the amounts
+  corresponding to agreement.
+
+- **Specter Desktop adds batching:**
+  [Specter Desktop][specter github], the hardware wallet multisig-focused GUI
+  for Bitcoin Core, added the ability to send transactions to multiple receivers.
+
+- **Lightning Labs releases Lightning Terminal:**
+  [Lightning Terminal][lightning terminal blog] is a visual, browser-based tool
+  for LN channel management, initially focusing on [Lightning Loop][news39 lightning loop announced].
+
+- **Wasabi adds support for PayJoin:**
+  [Wasabi 1.1.12][] adds support for the [BIP78][] PayJoin specification that
+  also works over Tor.
+
+- **BlueWallet for Desktop alpha announced:**
+  BlueWallet announced an [alpha desktop version][bluewallet desktop] for macOS
+  of their Lightning and Bitcoin wallet supporting bech32, hardware wallets, PSBTs,
+  watch-only, and more.
+
+- **BitcoinIsSafe.com lists Bitcoin software marked as malicious by antivirus products:**
+  The [bitcoinissafe.com][] website tracks detection rate of Bitcoin software
+  including Bitcoin Core, Electrum, and Wasabi within popular antivirus
+  products. The website also provides contact information for notifying
+  antivirus vendors about potential false positives.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.0-beta.rc4][lnd 0.11.0-beta] is a release candidate for the
+  next major version of this project.  It allows accepting [large
+  channels][topic large channels] (by default, this is off) and contains
+  numerous improvements to its backend features that may be of interest
+  to advanced users (see the [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19658][] modifies the `getnodeaddresses` RPC to allow returning
+  all known addresses when 0 is specified as the number of addresses requested.
+  Previously, there was a limit on the maximum number of addresses returned
+  mostly due to an internal implementation quirk.
+
+- [Bitcoin Core #18654][] adds a new `psbtbumpfee` RPC that takes the
+  txid of a transaction currently in the local node's mempool and
```

Comment:
Perhaps helpful to note this is for RBF signaling transactions that are in the wallet as well.